src/shared/components/iframe-runtime.tsx

```diff
@@ -101,13 +101,15 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  }, [url, authoredState, report, id, navImageUrl, navImageAltText]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  },[]);
 
+  const iframeStyle = wrapper === "scaler" ? iframeStyling : {width: "100%", height: iframeHeight, border: "none"};
```

Comment:
I'd recommend ignoring the `wrapper` here and just doing something like:
`const iframeStyle = iframeStyling ?? {width: "100%", height: iframeHeight, border: "none"}`

So that way if iframeStyling is set, then it is used, otherwise a default style is used.

Perhaps this means the scaling/fullscreen interactive needs to always be sure to send an iframeStyling prop, but as far as I can tell it is already doing this.